### ISSUE TITLE
Warning when used with PHP Artisan Tinker

### ISSUE THREAD

**Issue by mikeerickson** [getsentry/sentry-laravel#275]
When I am using Artisan Tinker, I receive notices such as

```bash
PHP Deprecated:  The Sentry/State/Hub::getCurrent() method is deprecated since version 2.2 and will be removed in 3.0...
```

For example, when using

```
>>> App\User::all();
```

I see a message 

```bash
PHP Deprecated:  The Sentry/State/Hub::getCurrent() method is deprecated since version 2.2 and will be removed in 3.0. Use SentrySdk::getCurrentHub() instead. in /Users/mikee/Documents/code/honu.salon/vendor/sentry/sentry/src/State/Hub.php on line 199
```

Two questions:

1. Why is this even occuring as the command I am executing in Tinker is legitimate?
2. Why would the Sentry package be using a deprecated method?

**Comment by stayallive** [comment]
@mikeerickson this is fixed in `1.3.0` of the Laravel package.

A change was made in the base SDK to fix issues with the original implementation (see: getsentry/sentry-php#847).

**Comment by stayallive** [comment]
Also it looks like I made a mistake: getsentry/sentry-laravel#276 this will be released as `1.3.1` later today.

**Comment by stayallive** [comment]
`1.3.1` was just released and should correctly fixed the issue this time 😄 

Thanks for your report!
